templates/pages/compare.html

```diff
@@ -14,7 +14,7 @@ <h1 class="page-heading">{{lang 'compare.header' products=comparisons.length}}</
                     <article class="card">
                         <figure class="card-figure">
                             <div class="card-img-container">
-                                <img class="card-image lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage image 'product_size' (cdn theme_settings.default_image_product)}}" alt="{{image.alt}}" title="{{image.alt}}">
+                                <img class="card-image lazyload" data-sizes="auto" src="{{cdn 'img/loading.svg'}}" data-src="{{getImage image 'product_size' (cdn 'img/ProductDefault.gif')}}" alt="{{image.alt}}" title="{{image.alt}}">
```

Comment:
@nnandani instead of hardcoding the file path here. Can we update `default_image_product` in `config.json` to use the right path.

Reply:
Default image product path not working that's why I have added this path for solution.